src/pages/MemberSettingsPage/MemberSettingsForm/index.jsx

```diff
@@ -36,21 +56,40 @@ const MemberSettingsForm = () => {
   const schema = yup.object().shape({
     firstName: yup.string().required("First name is required"),
     lastName: yup.string().required("Last name is required"),
+    email: yup.string().required("email is required"),
     phone: yup.string().required("phone number is required"),
     address1: yup.string().required("address1 is required"),
     address2: yup.string().required("address2 is required"),
+
   })
 
   const formik = useFormik({
     initialValues: {
       firstName: "",
       lastName: "",
+      email: "",
       phone: "",
       address1: "",
       address2: "",
-      email: "",
```

Comment:
Lets add in the initial value if they are already inputted that way they are shown to the user on first load